content/en/platform/corda/5.0-alpha/deploying/troubleshooting/support-bundles.md

```diff
@@ -9,3 +9,32 @@ menu:
 section_menu: corda-5-alpha
 ---
 <!--https://r3-cev.atlassian.net/browse/CORE-7232-->
+
+In the event of an issue, you need to know what information R3 requires from you to be able to perform problem determination. This collection of information is referred to as a support bundle. The following document lists the steps to troubleshoot a Corda deployment in Kubernetes and collect the support material.
```

Comment:
Seems a bit wordy.. Maybe something like:
This page describes how to gather the information required by R3 to assist you with troubleshooting in the event of an issue with a Kubernetes deployment.

We probably need a general team consensus as to whether we use document/section/page..

Reply:
Agree. Updated the short desc.